translations/de.json

```diff
@@ -26,7 +26,9 @@
 	"NNW": "NNW",
 
 	"FEELS": "Gefühlt {DEGREE}",
-	"PRECIP": "Niederschlagswahrscheinlichkeit",
+	"PRECIP": "Precipitation",
+	"PRECIP_POP": "Niederschlagswahrscheinlichkeit",
+	"PRECIP_AMOUNT": "Precipitation",
```

Comment:
"Niederschlagsmenge" 

(we need shorter words for the german stuff OR need to do a new layout where precipation is displayed, currently its in one line with feelslike an d takes too much space there) but thats another issue ;-)

Reply:
I agree, we have the same problem in Norwegian. As a possible workaround I added a `<span>`-element around it so it can be easily hidden with CSS.